apps/voting/app/src/script.js

```diff
@@ -223,17 +223,24 @@ async function loadVoteDescription(vote) {
     return vote
   }
 
-  const path = await app.describeScript(vote.script).toPromise()
-  vote.description = path
-    ? path
-        .map(step => {
-          const identifier = step.identifier ? ` (${step.identifier})` : ''
-          const app = step.name ? `${step.name}${identifier}` : `${step.to}`
-
-          return `${app}: ${step.description || 'No description'}`
-        })
-        .join('\n')
-    : ''
+  try {
+    const path = await app.describeScript(vote.script).toPromise()
+
+    vote.description = path
+      ? path
+          .map(step => {
+            const identifier = step.identifier ? ` (${step.identifier})` : ''
+            const app = step.name ? `${step.name}${identifier}` : `${step.to}`
+
+            return `${app}: ${step.description || 'No description'}`
+          })
+          .join('\n')
+      : ''
+  } catch (error) {
+    console.log('Error describing vote script', error)
```

Comment:
Should this be our function `log`? (the one that only logs in dev)

Reply:
Actually this should be `console.error`, it appears to be standard practice to use it. 

Logging for debugging is a different story.  We probably want to introduce a lean logger at some point